### ISSUE TITLE
Feedback on sound for GFL:B

### ISSUE THREAD

**Issue by Ashton-Morris** [phetsims/gravity-force-lab-basics#111]
@kathy-phet @ariel-phet @emily-phet @arouinfar 

The sound for GFL:B is at a good stage and we are looking for feedback on the current prototype.

Please take a look at [this version](https://phet-dev.colorado.edu/html/phet-io-wrapper-sonification/1.0.0-dev.107/phet-io-wrapper-sonification/gravity-force-lab-basics/html/gravity-force-lab-basics-sonification.html?sonificationFile=sonificationOptions&massSound=0&forceSound=0&massBoundaryLimitSound=2&accessibility), and let me know if you have any concerns about the sound, or any preferences regarding sounds on by default vs available as an enhanced sound.

**Comment by arouinfar** [comment]
Thanks @Ashton-Morris! I downloaded the dev version to review on my flight back home tomorrow.

**Comment by arouinfar** [comment]
Sorry for the delayed response! It's been a crazy few weeks for me.

@Ashton-Morris @jbphet @emily-phet these sounds are so fun! I think the meaning of the sounds was immediately clear, and students will have a lot of fun with them. I even found myself trying to compose a tune or two. :)

I tested with over-the-ear headphones, and these are the sounds I observed:
* Hum representing force: higher pitch = larger force
* Drumming sound representing mass: lower pitch = greater mass
* Click/thump representing extremes in position
* Standard checkbox and resetAllButton sounds

 The only suggestion I have would be a slight adjustment to the relative volumes. The checkbox and resetAllButton sounds seemed quite a bit louder than the other sounds, so I would dial those down a bit. Otherwise, I found the balance to be excellent.

**Comment by arouinfar** [comment]
Just to add -- I wouldn't make a standard/enhanced distinction here. I think all of these sounds should be included by default.

**Comment by ariel-phet** [comment]
@Ashton-Morris @arouinfar @emily-phet 

All seems pretty good...it took me a little while to verify that the drum beat related to mass changing was distinct from the force value changing.

My personal opinion would be to leave the sound related to force value for the "enhanced" sound. My reason is that the other sounds (like the drum beat, or the check box/reset all sounds) are less sustained than the force sound. Since the force sound is representing something that does not go away, For me, that creates a bit of difficulty...the force is changing but remains at a certain value. I think it is certainly able to be interpreted, but given the sustain of the sound it feels a little more something I have to "figure out" its meaning when using the sim, compared to the other sounds. 

Anyhow, I believe Kathy has before been of the opinion to go with full sound and see if we get push back...happy to be out voted, but for me having the force sound included is a bit on the overwhelming side.

**Comment by emily-phet** [comment]
@arouinfar @ariel-phet 
Thanks for your thoughts! Just jumping in here - @Ashton-Morris will probably respond as well - to ask for a little more elaboration on @ariel-phet's comment above. 

Are you saying that the short continuation of the force sound after interaction is the 'sustain' that you feel could interfere with the mapping of that sound to the force value/concept? Just wanted to clarify before suggesting we play around with that attribute. I can't recall off the top of my head what decision making went around particular attribute, but if we haven't before we could try shortening that and seeing how it feels.

**Comment by Ashton-Morris** [comment]
@arouinfar We did discuss changing the volumes of the reset and the checkbox sounds and will look into that in the next meeting, thanks.

**Comment by ariel-phet** [comment]
@emily-phet said

> Are you saying that the short continuation of the force sound after interaction is the 'sustain' that you feel could interfere with the mapping of that sound to the force value/concept?

Yes, that is correct. I don't necessarily think you need to change the force sound, but that would be my impetus for keeping it in the "enhanced" category. I think shortening it would make the bass sound of changing Mass and the force sound harder to distinguish. I can see why it was done the way it was, but I think it might be a little overwhelming for the casual user (or the one like me that still has some "on the spectrum" characteristics). Again, just my opinion.

**Comment by Ashton-Morris** [comment]
@arouinfar @ariel-phet We ended up editing the force sound in response to your comments. We looked at the ending duration of the force sound and altered it to be shorter but still longer than the duration of the mass sound, so they both can be distinguishable.

Since the concept of force is the most important aspect of the sim we feel the sound should be on by default. Since the beginning we have altered the volume and tone to make it more pleasurable to the ear and we haven’t gotten any negative feedback since. We also have two more interviews to review, where we’ll be checking again on the feedback for this sound. If we find something significant in these interviews, or later if we get pushback from the broader community, we will revisit it and make more adjustments.
